README.md

```diff
@@ -1,8 +1,8 @@
 # sonar-ruby
 
-[![Build Status](https://api.cirrus-ci.com/github/SonarSource/slang.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/slang)
-[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=org.sonarsource.slang%3Aslang&metric=alert_status)](https://sonarcloud.io/dashboard?id=org.sonarsource.slang%3Aslang)
-[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=org.sonarsource.slang%3Aslang&metric=coverage)](https://sonarcloud.io/component_measures/domain/Coverage?id=org.sonarsource.slang%3Aslang)
+[![Build Status](https://api.cirrus-ci.com/github/SonarSource/slang.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/sonar-ruby)
```

Comment:
```suggestion
[![Build Status](https://api.cirrus-ci.com/github/SonarSource/sonar-scala.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/sonar-ruby)
```

Reply:
Nice catch! It should be sonar-ruby though